### ISSUE TITLE
Training loss

### ISSUE THREAD

**Issue by FrauSong** [onwn/C2N#7]
Hi,

Thanks for sharing the code of this excellent paper. I am a little bit confused by Eq 4. Should this loss be backpropagated only to the discriminator? The second term of this equation differs from the original WGAN-GP. Could you be so kind as to give some hints on how to understand it?

Many thanks!
